onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -370,6 +370,13 @@ bool ConvNodeGroupSelector::Check(const GraphViewer& graph_viewer,
     return false;
   }
 
+  // Only per-tensor or per-output channel (axis == 1) quantization is supported
+  const auto& dq_attrs = dq_nodes[1]->GetAttributes();
+  if (const auto a_iter = dq_attrs.find("axis");
+      a_iter == dq_attrs.end() || a_iter->second.i() != 1) {
```

Comment:
The condition is not right. It should be "a_iter != dq_attrs.end() && a_iter->second.i() != 1".

Reply:
Thanks for the catch!